src/main/cli/cli.c

```diff
@@ -4731,6 +4731,46 @@ static void cliStatus(const char *cmdName, char *cmdline)
     }
 #endif
 
+#ifdef USE_GPS
+    cliPrint("GPS: ");
+    if (featureIsEnabled(FEATURE_GPS)) {
+        if (gpsIsHealthy()) {
+            cliPrint("connected, ");
+        } else {
+            cliPrint("NOT CONNECTED, ");
+        }
+        if (gpsConfig()->provider == GPS_MSP) {
+            cliPrint("MSP, ");
+        } else {
+            const serialPortConfig_t *gpsPortConfig = findSerialPortConfig(FUNCTION_GPS);
+            if (!gpsPortConfig) {
+                cliPrint("NO PORT, ");
+            } else {
+                cliPrintf("UART%d %ld(", (gpsPortConfig->identifier + 1), baudRates[getGPSPortActualBaudRateIndex()]);
+                if (gpsConfig()->autoBaud == GPS_AUTOBAUD_ON) {
+                    cliPrint("AUTO");
+                } else {
+                    cliPrintf("%ld", baudRates[gpsPortConfig->gps_baudrateIndex]);
+                }
+                cliPrint("), ");  // Place holder for queried config items to be added (firmware & protocol version, actual solution rate, etc.)
+            }
+        }
+        if (!gpsIsHealthy()) {
+            cliPrint("NOT CONFIGURED");
+        } else {
+            if (gpsConfig()->autoConfig == GPS_AUTOCONFIG_OFF) {
+               cliPrint("auto config OFF, ");
+            } else {
+                cliPrint("configured, ");
```

Comment:
This should be the end of the line?
```suggestion
                cliPrint("configured");
```

Reply:
Ah, yeah, I had it coded for adding more items. I'll clean that up in case we want to merge this first.